README.md

```diff
@@ -1,32 +1,95 @@
-# RebuildBlackBusiness.com
+<h1 align="center">Welcome to Rebuild Black Business 👋</h1>
+<p>
+  <img alt="Version" src="https://img.shields.io/badge/version-1.0.0-blue.svg?cacheSeconds=2592000" />
+  <a href="https://twitter.com/ThugDebugger" target="_blank">
+    <img alt="Twitter: ThugDebugger" src="https://img.shields.io/twitter/follow/ThugDebugger.svg?style=social" />
+  </a>
+</p>
 
-## Local Development
+> The website for https://www.rebuildblackbusiness.com/
 
-### 1. Clone the repository
+### 🏠 [Homepage](https://www.rebuildblackbusiness.com/)
 
-First, fork this repository by clicking the “fork” button at the top right. This creates your own copy of the repository where you’re able to make changes.
+## Mission
 
-Second, clone your copy of the repo to your local machine:
+## 🤝 Contributing
 
-```bash
-# clone the repo
-git clone git@github.com:YOUR_USERNAME_HERE/RBB-Website.git
+Contributions, issues and feature requests are welcome!<br />Feel free to check [issues page](https://github.com/Rebuild-Black-Business/RBB-Website/issues).
+
+### How can I get involved now?
```

Comment:
There's no mention of the Discord here. If the process is designed to require devs to go through Discord to get access, we should probably call out that process first.

Reply:
👍 - I'll add a paragraph here explaining the role Discord plays!